src/main/scala/com/cognite/spark/datasource/CdpRelation.scala

```diff
@@ -90,23 +90,20 @@ object PushdownUtilities {
       case EqualTo(colName, value) => PushdownFilter(colName, value.toString)
       case EqualNullSafe(colName, value) => PushdownFilter(colName, value.toString)
       case GreaterThan(colName, value) =>
-        PushdownFilter("min" + colName.capitalize, toMinTimeFormat(value))
+        PushdownFilter("min" + colName.capitalize, value.toString)
```

Comment:
Hmm, we're sure we're ok to remove the `toMin/MaxTimeFormat`? That stuff is tricky. Maybe it's less tricky nowadays with V1?

Reply:
Ok, looks like we're doing that still but in other places.

Reply:
Looks like this is actually not quite right, we're handling `<=` and `<` in the same way now. I'll fix it.